packages/cli/src/ui/components/views/SkillsList.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import type React from 'react';
+import { Box, Text } from 'ink';
+import { theme } from '../../semantic-colors.js';
+import { type SkillDefinition } from '../../types.js';
+
+interface SkillsListProps {
+  skills: readonly SkillDefinition[];
+  showDescriptions: boolean;
+}
+
+export const SkillsList: React.FC<SkillsListProps> = ({
```

Comment:
this is awesome. I want to add components like this for the other extension pieces so we can standardize the display!